app/models/like.rb

```diff
@@ -1,2 +1,10 @@
 class Like < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  belongs_to :post
+
+  after_commit :update_likes_counter, on: :create
+
+  def update_likes_counter
+    post.update(likes_counter: post.likes.count)
+  end
```

Comment:
- [ ] OPTIONAL
It's a good practice to mark the `update_likes_counter` method as private, when the `after_commit` method is implemented.